layouts/partials/home/project-info.html

```diff
@@ -1,5 +1,25 @@
 <section class="section">
   <div class="container has-text-centered box push-down">
+    <div class="columns is-vcentered is-variable is-20">
+      <div class="column">
+        <p class="title is-size-4 is-size-5-mobile">
+          Sign up for the latest "Longhorn Basics" Online Training.
```

Comment:
Imho doesn't need the inverted commas, it isn't implying anything. It is Longhorn Basics Online training. Maybe italics? @jillian-maroket thoughts?

Reply:
Agreed, @derekbit could you make these changes, please?

Reply:
Sure